### DISCUSSION TITLE
Send long arrays using FSK/LoRa

### DISCUSSION THREAD

**Discussion by radio-satellites** [jgromes/RadioLib#895] [Q&A]
I have a JPEG image that I would like to send via LoRa, captured using an esp32cam (`esp_camera_fb_get();`). However, RadioLib only provides and interface to send 256 bytes at a time. How would I go about splitting the image into 256-byte chunks to send? Does RadioLib provide a interface similar to `LRSendArray();` in other libraries?

Thank you!

**Comment by jgromes** [comment]
This is a basic C question that doesn't really have anything to do with RadioLib specifically. I suggest reading up on arrays and pointers; RadioLib provides a generic `transmit(uint8_t* data, size_t len)` method for all the supported transceivers, all you need to do is call it in loop.
